internal/server/v0/placestatvar/golden/get_place_stat_vars/santa_clara.json

```diff
@@ -3049,6 +3049,9 @@
         "Count_HousingUnit_2010To2014Date_OccupiedHousingUnit",
         "Count_HousingUnit_2010To2014Date_OwnerOccupied",
         "Count_HousingUnit_2010To2014Date_RenterOccupied",
+        "Count_HousingUnit_2010To2017Date_OccupiedHousingUnit",
```

Comment:
were there some census updates in this release too?

Reply:
There was subject table(S2502) CL submission which got built in the last UScensus run.